wallet/src/de/schildbach/wallet/ui/MainActivity.kt

```diff
@@ -435,12 +445,32 @@ class MainActivity : AbstractBindServiceActivity(), ActivityCompat.OnRequestPerm
         UnlockWalletDialogFragment.show(supportFragmentManager)
     }
 
+    private fun handleInvite(invite: InvitationLinkData) {
+        val acceptInviteIntent = AcceptInviteActivity.createIntent(this, invite, false)
+        startActivity(acceptInviteIntent)
+    }
+
+    override fun onUnlocked() {
+        if (pendingInvite != null) {
+            handleInvite(pendingInvite!!)
+            pendingInvite = null // clear the invite
+        }
+    }
```

Comment:
here we have the code to activate an invite after the user enters a PIN/fingerprint to dismiss the Lock Screen.